proxy/src/graphql/schema.rs

```diff
@@ -185,6 +253,22 @@ impl Query {
             .map(|id| juniper::ID::from(id.0.to_string()))
             .collect::<Vec<juniper::ID>>())
     }
+
+    fn identity(_ctx: &Context, id: juniper::ID) -> Result<Option<Identity>, error::Error> {
+        Ok(Some(Identity {
+            id: id.clone(),
+            shareable_entity_identifier: juniper::ID::new(format!("cloudhead@{}", id.to_string())),
+            metadata: IdentityMetadata {
+                handle: "cloudhead".into(),
+                display_name: Some("Alexis Sellier".into()),
+                avatar_url: Some("https://avatars1.githubusercontent.com/u/4077".into()),
+            },
+        }))
+    }
+
+    fn user(_ctx: &Context, handle: juniper::ID) -> Result<Option<juniper::ID>, error::Error> {
+        Ok(None)
+    }
```

Comment:
At some point we might need a flag here to mock that the user exists so that we can update the user profile (registered badge, remove the register user handle button) and search users and add them as members.

Reply:
I hope our mocks have evloved into proper endpoints by that point, otherwise we extend when the need arise.